adhocteam/Head-Start-TTADP
# add clam-av to docker-compose and env variables

This is a small pr to add env variables and clamav containers. 

**Description of change**
Adds the clamav containers to docker compose for local dev and adds env variables for the app to access it.

**How to test**
1. pull down changes and run `yarn docker:start`
2. Run `curl -F "name=test" -F "file=@path/to/local/file.txt" localhost:8081/scan`
3. Result should be "Everything ok : true"
4. Create a text file containing the string `X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*`
5. Run `curl -F "name=test" -F "file=@path/to/new/file.txt" localhost:8081/scan`
6. Result should be "Everything ok : false"


**Issue(s)**
https://github.com/HHS/Head-Start-TTADP/issues/204
